src/rules/group-exports.js

```diff
@@ -86,6 +95,18 @@ function create(context) {
         })
       }
 
+      // Report multiple `aggregated exports` from the same module (ES2015 modules)
+      Object.keys(nodes.sources)
```

Comment:
You can use the object.values package here, instead of having to use Object.keys.